### DISCUSSION TITLE
eÁFA M2M helyett...

### DISCUSSION THREAD

**Discussion by RuszinZsolt** [nav-gov-hu/eVAT#3] [Ötletek - gépi interfész]
Az eÁFA esetében is a lépcsőzetes bevezetés lenne a jó, ami a számla online adatszolgáltatásának is eddig jól működött. A lépcsőzetesség szükségességét valamilyen rejtélyes ok miatt az eÁFA kapcsán a NAV nem hajlandó még megfontolni se, ezért is fulladt kudarcba a többszörösen elhalasztott "ÁFA BEVALLÁS-TERVEZET" nevű projekt is. A mostani félig új elnevezéssel ezt a kudarcot igyekszik a hivatal takargatni.

_Sajnos az adóhatóság még mindig az egyszerűsíthetetlenül bonyolult ÁFA bevallást akarja automatizálni, ahelyett, hogy az alacsony bonyolultságú és ezáltal automatizálható részterületeket célozna meg és ha azok jól működnek már, akkor annak tapasztalatából építkezne tovább._

**Javasolható, hogy az újkatásokkal és az átalányadózást választókkal kezdjék a folyamatot.** 

Ehhez szükséges az érintett kört behatárolni:

1. Ki kell zárni az eÁFA bevallást élvezők köréből a pénzforgalmi áfát választókat.
2. Ki kell zárni az eÁFA bevallást élvezők köréből a különbözeti áfát választókat.
3. Ki kell zárni az eÁFA bevallást élvezők köréből az OSS használatára kötelezetteket.

További kizárási esetek is elképzelhetőek.

_**A fennmaradó körnek (kb. 400 ezer adózó) kínálják be a belföldi számlakibocsátás adatait az eÁFA bevallásba, az alanyi mentesekre is! Be kell még kínálni a közösségi adatszolgáltatásból eredő forgalmat is, a gép-gép kapcsolaton keresztül, ahol külön szoftver nélkül lehetne az áfabevallást benyújtottnak tekinteni.**_ 

**Erre várom a tisztelt érdeklődök és a tekintetes adóhatóság véleményét, álláspontját.**

**Comment by BlaskoEva** [comment]
Érdemes még az ÁFA arányosítással érintetteket is kizárni

Ezen kívül egységes megoldásra van szükség a jogszabály alapján csak részben visszaigényelhető ÁFA-k kezelésére  - telefonszámlák, nyílt végű lízingek, tartós bérletek, szgk javítási számlák, stb,- hogy csak néhány leggyakoribbat említsek a gyakorlatomból.

**Comment by RuszinZsolt** [comment]
17 nap és a NAV részéről még mindig nem érkezett reakció.

Jó lenne megérteni, hogy a hivatal miért kerüli, hogy legyen egy kisebb projekt, amin az eÁFA M2M kitanulható. A jogalkotó is - gondolom - örömmel idomulna egy ilyen projekthez, mint egy újabb arcvesztéssel járó mega-giga-all-in projekt újabb kudarcához.

Értem, hogy kellemetlen témát feszegetek, már megint. Ezt csinálom 25 éve és egy sor dolgot sikerült is a hivatalban kívülről megváltoztatni.

Apropó, változtatás. Az EBEV ugye még marad egy pár évig, mert ugye az ÜPO úgy gond, ahogy van. Nemcsak hiányoznak belőle fontos a funkciók, de az egész koncepció ismét egy könyvelőellenes szemlélettel lett megvalósítva. Szeretnénk kapni egy könyvelőbarát portált, ami nem növeli, hanem csökkenti a kattintásszámot és végre jó lenne kinyitni a gép-gép kapcsolatokat, apránként, szorgalmasan. Lehetne kezdeni például az adófolyószámla gép-gép kapcsolaton keresztüli lekérdezésével.

**Comment by NTCA-tax** [comment]
A GitHub-on publikált tervezet kizárólag a gép-gép kapcsolatról szól. Bár teszünk említést az áfabevallás kiajánlásáról (pl. 5. oldal), de ez a dokumentum nem az, ahol leírjuk, hogyan fog az működni. De nyilvánvaló módon van kapcsolata a kettőnek egymáshoz. 

Nem akarjuk megmondani egyetlen adózónak sem, hogy neki márpedig kizárólag az egyik vagy másik megoldást KELL választania. Sőt az M2M rendszerben próbálunk minél több adatot odarakni, és majd az adózó szoftverfejlesztője emelje be a könyvelő programjába oda és amikor ő szeretné, de dönthet úgy is, hogy nem foglalkozik ezekkel. Nem mi ismerjük a legjobban azt, hogy milyen adatra és mikor lenne szüksége egy könyvelőnek, erről is szól ez a véleményezés. Amennyiben felmerül olyan javaslat, hogy máshogy vagy máskor adjuk bizonyos adatokat, akkor itt van a helye és ideje ezeknek. Itt még nem kész specifikációról beszélünk, hanem egy olyan tervezetről, amit az érintetteknek megmutatunk véleményezésre.  A véleményeket összegezzük, és annak alapján haladunk tovább.

Kérem, hogy ne hozzunk ebbe a beszélgetésbe az M2M-től eltérő elképzelések megvitatását, mert annak nem ez a felülete. Itt alapvetően javaslatokat, véleményeket várunk a publikált tervezettel kapcsolatban.

Amit nem teljesen értek, az a következő mondat: "Be kell még kínálni a közösségi adatszolgáltatásból eredő forgalmat is, a gép-gép kapcsolaton keresztül, ahol külön szoftver nélkül lehetne az áfabevallást benyújtottnak tekinteni." Ez a javaslat pontosan mit takar? Ez egyértelműen M2M javaslatnak látom, de ezt jobban ki tudná fejteni?

Az adófolyószámla egyébként benne van az elképzeléseinkben (lásd 2.4) és köszönjük eddig is az ezzel kapcsolatos sok javaslatot különböző szervezetektől.  Azért nem vontuk be ebbe a dokumentumba, mivel jelenleg az látszik, hogy más projekt fogja megvalósítani az adóhivatalon belül.

**Reply by RuszinZsolt** [reply]
Kedves Ismeretlen hozzászóló (NTCA-tax, azki?). Az a baj, hogy a NAV megy a saját feje után, éppen a koncepcionális döntésekből hagyja ki a könyvelőket. Ezek után jön, hogy akkor "ne hozzuk be a beszélgetésbe"? Ez milyen hatalmi módszer már? Miért kell ilyesmiket írni pont a Githubon?

Továbbra is fenntartom (fenntartjuk), hogy az eÁFA M2M módszereit a legkisebb adózói körre kell kipróbálni és onnan fejlődni. A NAV hibát követ el, ha túl széles körre feszíti ki a fejlesztést, ahogy ez az ÁFA bevallástervezet kudarca is mutatja (apropó, az mennyi közpénzt emésztett fel?).

Mire gondoltunk a közösségi áfa adatok kapcsán?

A NAV egy bő évtizede egész jó minőségben hozzájut a közösségi forgalom adataihoz, de ezeket egyáltalán nem osztja meg az adózókkal, csak ráül és kész. Tapasztalataink szerint a havi közösségi forgalmak 80%-a egyetlen számlából erednek (tagországonként). Ez főleg a kisebb adózóknál van így. Az összesített forgalom tehát egyetlen bizonylatra vonatkozik. Ezt használni kellene az M2M kommunikációban, azt be kellene kínálni az adózóknak. Miután a javaslatunk éppen a legkisebb adózók körére szólt, ezért náluk ez fontos és hasznos tétel lenne.

Visszatérve az alap-egyeztetési kérdésre: A hanyatló Nyugaton ez a fajta kapcsolattartás nem így működik, főleg a koncepcionális döntések terén. Sokkal több az érdemi kommunikáció, a kormányzati akarat pedig szívesebben követi a valós igényeket és nem öncélúan fejleszt, közpénzből. Az elmúlt 10 évben az ezzel kapcsolatos helyzet Magyarországon sokat romlott. 

Na tessék, idehoztam a témát, ráadásul egy ismeretlennek írogatok. Most mi lesz? Házkutatás lesz nálam, megint?

**Comment by tniLAB** [comment]
Nézze Zsolt, ilyen típusú hozzászólásokkal a valódi javaslattevőktől (nem feltétlenül magamról beszéltem) veszi el a NAV fejlesztőinek az idejét.
De, hogy egy megoldást is javasoljak az egyik felvetett problémájára: bármelyik ügyfélszolgálati kirendeltségre ha bemegy, akkor ott bemutatkozás után nagyon jó offline beszélgetéseket lehet tartani a felek vezeték- és keresztneveinek teljes tudatában.
Szép napot,
Bátonyi Bálint

**Reply by RuszinZsolt** [reply]
> Nézze Zsolt, ilyen típusú hozzászólásokkal a valódi javaslattevőktől (nem feltétlenül magamról beszéltem) veszi el a NAV fejlesztőinek az idejét. De, hogy egy megoldást is javasoljak az egyik felvetett problémájára: bármelyik ügyfélszolgálati kirendeltségre ha bemegy, akkor ott bemutatkozás után nagyon jó offline beszélgetéseket lehet tartani a felek vezeték- és keresztneveinek teljes tudatában. Szép napot, Bátonyi Bálint

A hivatali cinizmus itt nem sokat segít. Viszont továbbítani lehetne az ÁFA  bevallás-tervezet értelmi szerzőinek a fenti diskurzust, hátha végre reagálnak, érdemben. A továbbítás néhány másodpercet vesz igénybe. Ez a sokmilliárdot eltapsoló Pillér Kft.-nél és a Pillér Kft-től bármit (is) "készként" átvevő NAV IFO-nál nem lehet gond.

**Comment by RuszinZsolt** [comment]
Sikerült már megfejteni, hogy mit jelent az, hogy "Be kell még kínálni a közösségi adatszolgáltatásból eredő forgalmat is, a gép-gép kapcsolaton keresztül"? 

Segítek: a NAV számára utólagosan, de rendelkezésre áll ezt az adat, közösségi adószám alapján leadott időszaki forgalomként, ha jól tudom, akkor a frankfurti központból. Ezeket az adatokat a NAV elvileg rendszerezheti és visszaadhatja az M2M interfészen keresztül, ha akarja. Ez nagyjából akarat és egy fordítóprogram, valamint tárolókapacitás kérdése. Menni fog?

**Reply by omachtandras** [reply]
A kérdés az, hogy mikor áll rendelkezésre. Ha azonnal, akkor ez egy tök jó megoldás lenne. Ha egy-két hónap múlva, akkor sajnos már semmit nem ér. :(
Van (valakinek) arról információja, hogy mikor állnak rendelkezésre ezek az adatok a NAV oldalán?

**Comment by NTCA-tax** [comment]
Azért könnyű kilogikázni, hogy egy bevallásban szereplő adat az adóhivatalnak csak a bevallást követően áll a rendelkezésére (még akkor is ha más tagállami bevallásról beszélünk). Tehát a bevallás készítéshez ezt nem lehet használni, ráadásul negyedéves az adatfrissesség. A VIES adatokkal további probléma, hogy ilyen célra sem jogilag, sem technikailag nem használható. Nem véletlenül van a ViDA-ban a DRR...

**Comment by Macskafarka** [comment]
A Héa irányelv szerint az adómegállapítási időszak tagállami döntés szerint lehet 1, vagy 2, vagy 3 hónap, sőt lehet akár 1 év is. Továbbá az adómegállapítási időszak végétől számított tagállami döntéstől függően, de legfeljebb 2 hónapon belül kell az adóalanynak benyújtani a saját adóbevallását. És csak azt követően adhatnak át egymásnak adatot az állami adóhatóságok.

**Reply by omachtandras** [reply]
Nem a bevallási adatokat kell átadni, hanem az online számlához hasonló rendszerekbe beküldött adatokat, ahogy most is vevő oldalon le lehet tölteni a számla adatokat már akkor, amikor még a szállító be sem vallotta. De, majd lehet, hogy tényleg a DRR lesz a megoldás, meglátjuk.

**Comment by Macskafarka** [comment]
Azonban ha jól tudom, akkor mostanában épp mozgolódása van a közösségi áfa bevallással kapcsolatban az unión belül...

**Comment by NTCA-tax** [comment]
@Macskafarka : Itt tudsz a "mozgolódásról" többet olvasni: https://ec.europa.eu/info/law/better-regulation/have-your-say/initiatives/13186-VAT-in-the-digital-age_en

**Comment by Macskafarka** [comment]
@NTCA-tax  Köszönöm!
